resources/lang/en/thoughts/alive/general.json

```
@@ -1420,12 +1420,12 @@
             "Is wondering if {PRONOUN/m_c/subject} {VERB/m_c/are/is} good enough...",
             "Is thinking about where {PRONOUN/m_c/subject} {VERB/m_c/belong/belongs}",
             "Wonders if other cats laugh with {PRONOUN/m_c/object} or at {PRONOUN/m_c/object}...",
-            "Is trying to figure out whether {PRONOUN/m_c/subject} {VERB/m_c/do/does}n't understand something, or {PRONOUN/m_c/poss} lecturing Clanmate doesn't",
+            "Listens to r_c's lecture, unsure whether {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} right",
```

```suggestion
            "Listens to r_c's lecture, unsure of whether or not {PRONOUN/r_c/subject}{VERB/r_c/'re/'s} right",
```
this one is more of a purrsonal nitpick, if you prefer how your version flows feel free to keep it!

---

I think they're both fine, but I also think shorter is better so I think I'll leave it as-is! ty for the edits